tests/timeseries/test_fill_missing_timestamps.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import pandas as pd
+from janitor.timeseries import fill_missing_timestamps
+
+
+@pytest.mark.timeseries
+def test_datatypes_check():
```

Comment:
I'd like to suggest parametrizing the test. The full docs on how to do so are available [here](https://docs.pytest.org/en/stable/parametrize.html), but here's a lightweight example.

```suggestion
@pytest.mark.parametrize(
    "df,frequency,column_name,first_time_stamp,last_time_stamp",
    [
        (4, '1T', None, None, None),
        (pd.DataFrame([]), 2, None, None, None),
        (pd.DataFrame([]), '1B', 1, None, None),
        (pd.DataFrame([]), '1B', 'DateTime', '2020-01-01', None),
        (pd.DataFrame([]), '1B', 'DateTime', pd.Timestamp(2020, 1, 1), '2020-02-01'),
]
def test_datatypes_check(df, frequency, column_name, first_time_stamp, last_time_stamp):
```

In doing so, you can make the test block:

```python
with pytest.raises(TypeError):
    fill_missing_timestamps(df, frequency, column_name, first_time_stamp, last_time_stamp)
```